### DISCUSSION TITLE
migrating to new computer

### DISCUSSION THREAD

**Discussion by e-piping** [moneymanagerex/moneymanagerex#5456] [Q&A]
Hi there,
i need to do a clean install of my os, so i'll need to move my files over manually. what files or folders should i move to retain my preferences and data, and where will i find them or at least what should i search for?

**Comment by tactilis** [comment]
Your database is the `.mmb` file (or `.emb` file if you have encrypted it).

The MMEX Options and other saved settings are in the `mmexini.db3` file
